soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnsz

```diff
@@ -23,4 +23,8 @@ config EEPROM_XEC
 	default y
 	depends on EEPROM
 
+config COUNTER_XEC
+	default y
+	depends on COUNTER
+
```

Comment:
This can be removed following 0f1ed0f4a05

Reply:
Yes, it can be removed.
The patch is updated. Thanks to review